docs-requirements.in

```diff
@@ -19,3 +19,6 @@ exceptiongroup >= 1.0.0rc9
 
 # See note in test-requirements.in
 immutables >= 0.6
+
+# types used in annotations
+pyOpenSSL
```

Comment:
Maybe update the comment in `test-requirements` too?

Reply:
it's not just used for annotations in tests, the code uses it, but it wasn't needed in docs previously until it showed up in the annotations.

Reply:
Ah, I meant update the comment to add that it's used in annotations too.